torch/export/dynamic_shapes.py

```diff
@@ -898,6 +898,7 @@ def _create_static_dim(tensor, i, value):
             return _StaticDim(str(value), (int,), {"value": value})
 
         # clean out decorators from user side, or previous export call
+        # we also delete these attributes in non_strict_utils.py/make_constraints()
```

Comment:
maybe we could have some context manager for handling setting these attrs and then removing these attrs?